index.html

```diff
@@ -2120,588 +2138,378 @@ <h3>Thing Root Object</h3>
+    ],
+    "title": "MyThing",
+    "description": "Human readable information.",
     ...
     "properties": {
         "on": {
+            "title" : "On/Off",
             "type": "boolean",
             "forms": [...]
         },
         "status": {
+            "title" : "Status",
             "type": "object",
-            "properties": {
-                "brightness": {
-                    "type": "number",
-                    "minimum": 0.0,
-                    "maximum": 100.0
-                    },
-                    "rgb": {
-                    "type": "array",
-                    "items" : {
-                        "type" : "number",
-                        "minimum": 0,
-                        "maximum": 255
-                    },
-                    "minItems": 3,
-                    "maxItems": 3
-                }
-            },
-            "required": ["brightness", "rgb"],
+            ...
             "forms": [...]
         }
     },
     ...
+}
 </pre>
-</aside>
-
-</section> <!-- end of id="property-serialization-json"-->
-
-<section id="action-serialization-json">
-<h3><code>actions</code></h3>
-
-  <p>
-    In a <code>Thing</code> instance, the value assigned to <code>actions</code>
-    is a <a>Map</a> of instances of <code>ActionAffordance</code>.
-    <span class="rfc2119-assertion" id="td-actions">
-    All name-value pairs of a <a>Map</a> of <code>ActionAffordance</code> instances
-    MUST be serialized as members of the JSON object that results from serializing the <a>Map</a>;
-    the name of a pair MUST be serialized as a JSON string and the value of the pair, an
-    instance of <code>ActionAffordance</code>, MUST be serialized as a JSON object.
-  </span></p>
-
-  <p><span class="rfc2119-assertion" id="td-action-names">
-    All name-value pairs of an instance of <code>ActionAffordance</code>,
-    where the name is a <a>Vocabulary Term</a> included in (one of) the <a>Signatures</a> of
-    <code>ActionAffordance</code> or <code>InteractionAffordance</code>,
-    MUST be serialized as members of the JSON object that results from
-    serializing the <code>ActionAffordance</code> instance, with the
-    <a>Vocabulary Term</a> as name.
-  </span></p>
 
   <p>
-  <span class="rfc2119-assertion" id="td-action-objects">
-    The values assigned to <code>input</code> and <code>output</code> in an instance of
-    <code>ActionAffordance</code>
-    MUST be serialized as JSON objects.
+  The JSON members named <code>titles</code> and <code>descriptions</code> are 
+  used within the TD document to provide human-readable metadata
+  in multiple languages within a single TD document.
+  <span class="rfc2119-assertion" id="td-multi-languages">
+    All name-value pairs of a <code>MultiLanguage</code> <a>Map</a>
+    MUST be serialized as members of a JSON object,
+    where the name is a well-formed language tag as defined by [[!BCP47]]
+    and the value the human-readable string.
```

Comment:
"the value is a human-readable string in the specified language".   The "is a" is an editorial grammar fix, the second part is a suggested addition.  It is obvious but because this is spec we should probably state it.   It should not be legal to say something is in Japanese and then given an English string...

Reply:
Clarified